Lib/test/test_binascii.py

```diff
@@ -368,8 +384,6 @@ def test_qp(self):
         self.assertEqual(b2a_qp(type2test(b'a.\n')), b'a.\n')
         self.assertEqual(b2a_qp(type2test(b'.a')[:-1]), b'=2E')
 
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
```

Comment:
Please break these lines out into a separate commit.

Reply:
I separated these lines in https://github.com/RustPython/RustPython/pull/4344/commits/dc9e958bb3172416e6d3c7747f32778c02436e13 commit. 🙏🏻